AWSPinpoint/AWSPinpointSessionClient.m

```diff
@@ -72,9 +72,7 @@ @interface AWSPinpointSessionClient()
 @end
 
 @interface AWSPinpointAnalyticsClient()
-
-- (void)removeAllGlobalCampaignAttributes;
-- (void)setCampaignAttributes:(NSDictionary*) campaign;
+- (void)removeAllGlobalFeatureAttributes;
```

Comment:
I know you didn't introduce this, but seeing this here is kind of disappointing that we don't have a clean callback interface from our low-level session client all the way back up to the analytics client.  I'd expect something like the analytics client to conform to a protocol that allows to be notified of its downstream session being ended but.. hm. yea.. okay..  i guess we can refactor this later.